MVP-UI/MVP-UI/Extension/UIFont+Extension.swift

```diff
@@ -0,0 +1,21 @@
+// UIFont+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Расширение доступа к шрифтам
+extension UIFont {
+    class func verdana(size: CGFloat) -> UIFont {
+        if let customFont = UIFont(name: "Verdana", size: size) {
+            return customFont
+        }
+        return UIFont.systemFont(ofSize: size)
+    }
+
+    class func verdanaBold(size: CGFloat) -> UIFont {
+        if let customFont = UIFont(name: "Verdana-Bold", size: size) {
+            return customFont
+        }
+        return UIFont.systemFont(ofSize: size)
+    }
```

Comment:
```suggestion
    class func verdanaBold(size: CGFloat) -> UIFont {
        if let customFont = UIFont(name: "", size: size) {
            return customFont
        }
        return systemFont(ofSize: size)
    }
    
    private func systemFont(ofSize: CGFloat) -> UIFont {
         UIFont.systemFont(ofSize: size)
    }
```
раз вызов системного используется более чем в 1 месте
его надо вынести